GUI/src/cz/fidentis/gui/NavigatorTopComponent.java

```diff
@@ -480,4 +492,30 @@ public void clearSelectionIfNeeded(ProjectTopComponent project) {
             }
         }
     }
+
+    /**
+     * Creates a copy of given list of FacialPoints reversely transformed by given
+     * transformations. This is needed when displaying original FacialPoints after
+     * models were already registered by the transformations.
+     * @param points facial points of a model
+     * @param trans transformations used to the model
+     * @return copy of facial points reversely transformed by transformations.
+     */
+    private List<FacialPoint> applyReverseICPTrans(List<FacialPoint> points, List<ICPTransformation> trans) {

```

Comment:
Bolo by dobre to hodit do nejakej samostatnej triedy spolu s dalsimi reversami (pravdepodobne do Processing modulu)